packages/frontend/src/components/common/ResourceView/ResourceAttributeCount.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Flex, Text, Tooltip, useMantineTheme } from '@mantine/core';
+import { type Icon as IconType } from '@tabler/icons-react';
+
+const ResourceAttributeCount: React.FC<{
+    Icon: IconType;
+    count: number;
+    name?: string;
+}> = ({ Icon, count, name }) => {
+    const theme = useMantineTheme();
+    const icon = <Icon color={theme.colors.gray[6]} size={14} />;
```

Comment:
check out `MantineIcon`

you can do stuff like
`<MantineIcon icon={IconShare2} color="gray" />`